src/graph/graph.c

```diff
@@ -233,6 +279,13 @@ Graph *Graph_New(size_t node_cap, size_t edge_cap) {
     assert(pthread_mutex_init(&g->_mutex, NULL) == 0);
     assert(pthread_mutex_init(&g->_writers_mutex, NULL) == 0);
 
+    // Create edge accumulator binary function
+    if(!_graph_edge_accum) {
+        GrB_Info info;
+        info = GrB_BinaryOp_new(&_graph_edge_accum, _edge_accum, GrB_UINT64, GrB_UINT64, GrB_UINT64);
+        assert(info == GrB_SUCCESS);
+    }
```

Comment:
I think it would be preferable to build the accumulator with one call from `RedisModule_OnLoad`.

Reply:
I don't want to mix Graph internals and module initialisation.